content/blog/2019-05-14-nasapower.md

```diff
@@ -0,0 +1,625 @@
+n <- length(unique(regional_t2m$ANN))
+
+# plot the annual average temperature using viridis
+plot(T2M_ann_regional, col = viridis(n = n))
+```
+
+![](/img/blog-images/2019-05-14-nasapower/regional-t2m-1.png)<!-- -->
+
+As you can see, because the data are georeferenced it is easy to use
+them in R’s spatial packages including sf and raster. [Emerson Del
+Ponte](https://delpontelab.netlify.com) used the data in a talk at the
+International Congress of Plant Pathology in August 2018, “[Can Rainfall
+be a Useful Predictor of Epidemic Risk Across Temporal and Spatial
+Scales?](https://speakerdeck.com/emdelponte/can-rainfall-be-a-useful-predictor-of-epidemic-risk-across-temporal-and-spatial-scales)”,
+see slides
+[23](https://speakerdeck.com/emdelponte/can-rainfall-be-a-useful-predictor-of-epidemic-risk-across-temporal-and-spatial-scales?slide=23),
+[24](https://speakerdeck.com/emdelponte/can-rainfall-be-a-useful-predictor-of-epidemic-risk-across-temporal-and-spatial-scales?slide=24)
+and
+[25](https://speakerdeck.com/emdelponte/can-rainfall-be-a-useful-predictor-of-epidemic-risk-across-temporal-and-spatial-scales?slide=25)
+for maps created using nasapower and the POWER data for two states in
+Brazil. These maps took a bit more work to query the API and generate,
+but I plan to add an example vignette detailing how this can be done in
+a future release.
+
+### Conclusion
+
+Even though the package took me well over one year to write and work out
+all of the bugs and has only three functions, I learned quite a bit from
+the experience. The new API really does improve the ability for a
+developer to write a client to interface with the data. Now the
+nasapower package is able to access all of the data available whereas
+the initial version was only able to work with the “AG” community data.
+
+While nasapower does not redistribute the data or provide it in any
+way, if you use the data, we encourage users to follow the requests of
+the POWER Project Team.
+
+> When POWER data products are used in a publication, we request the
+> following acknowledgement be included: “These data were obtained from
+> the NASA Langley Research Center POWER Project funded through the NASA
+> Earth Science Directorate Applied Science Program.”
+
+The approach I have used mostly works, but there has been at least one
+example where data are listed as being available in the POWER database
+but querying by the apparently proper `pars` does not work, see [Issue
+\#34](https://github.com/ropensci/nasapower/issues/34). Because of
+issues like this, in what appear to be edge cases, I suggest checking
+the web interface if you experience difficulties querying data.
+Hopefully the API itself has settled out a bit and will not have the
+sudden changes that I experienced. The POWER team have been supportive
+of the package, and I have received feedback and interaction from other
+users along the way that helped me by finding and reporting bugs but
+most of all the [rOpenSci review
```

Comment:
maybe put this part in a separate sentence? "...but most of all the rOpenSci review process ..."